### PR TITLE
Fix HdfsEnv::UnSchedule() API error

### PR DESCRIPTION
**Pull request by xh931076284**
Fix bug in UnSchedule() function requested a return-value and add function to set and get the delay_write_rate option

### CODE REVIEW

**Comment by IslamAbdelRahman** [comment]
Thanks @xh931076284, Travis java build is failing .. can you please take a look

**Comment by xh931076284** [comment]
ok

File: java/rocksjni/options.cc [hunk 1]
```diff
@@ -4463,6 +4463,20 @@ jlong Java_org_rocksdb_DBOptions_writeThreadSlowYieldUsec(
...
+    reinterpret_cast<rocksdb::DBOptions*>(jhandle)->
+        delay_write_rate = static_cast<int64_t>(delay_write_rate);
+
+  }
+
+  jlong Java_org_rocksdb_DBOptions_delayedWriteRate(
+    JNIEnv* env, jobject jobj, jlong jhandle){
+
+      return reinterpret_cast<rocksdb::DBOptions*>(jhandle)->
+          delay_write_rate;
```
Comment (IslamAbdelRahman) [inline java/rocksjni/options.cc]: This is the error, the option is `delayed_write_rate` instead of `delay_write_rate`

Comment (xh931076284) [inline java/rocksjni/options.cc]: @IslamAbdelRahman I have checked it and changed it.

**Comment by IslamAbdelRahman** [comment]
Thanks @xh931076284 for your contribution !

**Pull request merged**